programming/typescript/challenges/index.js

```
@@ -9,11 +9,39 @@ console.log('Greeting result:', helloAll);
+ * Challenge 26: Difference of Squares
+ * Demonstrates the difference of squares functionality
+ */
+var squares = new Challenges.Squares(5);
+console.log('Square of sum:', squares.squareOfSum);
+console.log('Sum of squares:', squares.sumOfSquares);
+console.log('Difference:', squares.difference);
+/**
+ * Challenge 27: Gigasecond
+ * Demonstrates the gigasecond functionality
+*/
+var gigasecond = new Challenges.Gigasecond(new Date('2013-01-07'));
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole module. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)